blockchain/tx_pool.go

```diff
@@ -641,6 +647,14 @@ func (pool *TxPool) Pending() (map[common.Address]types.Transactions, error) {
 	return pending, nil
 }
 
+func (pool *TxPool) UnlockedPending() (map[common.Address]types.Transactions, error) {
```

Comment:
Is it safe to read pending without a lock?

Reply:
Yes, this is always called after txmu locks.

Reply:
Please make a comment that this should be called after acquiring lock.

Reply:
https://github.com/kaiachain/kaia/pull/309/commits/33b3a096ce788fe0c2a4ce19b8f465c2cc597005